atlas_densities/app/cell_densities.py

```diff
@@ -797,12 +797,13 @@ def fit_average_densities(
     with open(config["realignedSlicesPath"], "r", encoding="utf-8") as file_:
         slices = json.load(file_)
 
+    toLower = "Layer "
     with open(config["cellDensityStandardDeviationsPath"], "r", encoding="utf-8") as file_:
         cell_density_stddev = json.load(file_)
         cell_density_stddev = {
             # Use the AIBS name attribute as key (this is a unique identifier in 1.json)
             # (Ex: former key "|root|Basic cell groups and regions|Cerebrum" -> new key: "Cerebrum")
-            region_name.split("|")[-1]: stddev
+            region_name.split("|")[-1].replace(toLower, toLower.lower()).replace("/layer", ", layer"): stddev
```

Comment:
not the biggest fan of making changes to names like this, but the `std_cells.json` is already a strange format (encoding a tree in a string?).

Ideally, that file would be updated to:
1) be a tree
2) use acronyms instead of human readable names

Reply:
Me neither I don't like such ad-hoc changes but I don't see another way unless enforcing a standard naming at files creation.
We just need this code to work if we want people to run the pipeline smoothly.